workflows/tomte.nf

```diff
@@ -204,7 +209,18 @@ workflow TOMTE {
     )
     ch_versions = ch_versions.mix(IGV_TRACKS.out.versions)
 
-/*
+    ch_pedfile = CREATE_PEDIGREE_FILE(ch_samples.toList()).ped
```

Comment:
What if sex is not given?

Reply:
I think the sex will be assigned as "unknown" and still predicted by Peddy. But I'll give it a testrun and see what happens

Reply:
Trying it out without sex column:

```
sample,case,fastq_1,fastq_2,strandedness                                                                                                          
hg002_rna_15m,hg002_rna_case_15m,/data/hg002_15M_R1.fastq.gz,/data/hg002_15M_R2.fastq.gz,reverse
hg002_rna_30m,hg002_rna_case_30m,/data/hg002_30M_R1.fastq.gz,/data/hg002_30M_R2.fastq.gz,reverse
```

Runs fine with Peddy, yielding two sex check files as such:

```
[jakob@rs-fe1 peddy]$ cat hg002_rna_15m.sex_check.csv
sample_id,ped_sex,hom_ref_count,het_count,hom_alt_count,het_ratio,predicted_sex,error
hg002_rna_15m,unknown,0,93,925,0.1005,male,True
```

Note that I translate everything non M or F into 0 over at the `create_pedigree_file.nf:

```
def sex_int = samples[i].sex == "M" ? "1" : samples[i].sex == "F" ? "2" : "0"
```